libnavigation-base/src/main/java/com/mapbox/navigation/base/options/OnboardRouterOptions.kt

```diff
@@ -65,8 +74,9 @@ class OnboardRouterOptions private constructor(
      */
     class Builder {
         private var tilesUri: URI = URI("https://api.mapbox.com")
-        private var tilesVersion: String = "2021_01_30-03_00_00"
+        private var tilesVersion: String = ""
         private var filePath: String? = null
+        private var keepOlderTilesVersions: Boolean = true
```

Comment:
```suggestion
        private var keepOlderTilesVersions: Boolean = false
```

Reply:
I think it's discussable. Should we keep older tiles by default? 

/cc @mapbox/navigation-android @etl @mskurydin

Reply:
Nah, the other way around 😅 

`keepOlderTilesVersion` `true` means keep it by default to me 😂 

Maybe it's a naming issue 😝

Reply:
I don't see a reason for us to keep them. If users are relying on our default fallback, they probably aren't interested in managing the tilesets anyway, so we should be good to evict it for them.